### DISCUSSION TITLE
traning set and test set

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#644] [Q&A]
Is deepmd kit DNN back propagation? Will the value of numb_test in input.josn file affect the model? Will increasing the value of numb_test reduce the test error in lurcve.out? When I run deepmd with only energy, the test error will not converge, and the training error will only decrease by an order of magnitude. 
Looking forward to your reply!

**Comment by marian-code** [comment]
No, `numb_test` does not affect the quality of the fit. `numb_test` is essentially `batch_size` for test set. The total number of test structures is determined solely by the size of last(test) set.

I do not have the answers to your other questions. But the poor quality of the fit certainly has nothing to do with `numb_test` setting.

**Comment by DingChangjie** [comment]
Did you include virial information (virial.npy) in your training set ? I also encountered similar problems. After included viral, things worked well and the error could reach a reasonable level.
